src/components/PDFView/index.js

```diff
@@ -283,7 +286,7 @@ class PDFView extends Component {
                     }) => this.setState({containerWidth: width, containerHeight: height})}
                 >
                     <Document
-                        error={<Text style={[styles.textLabel, styles.textLarge]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
+                        error={<Text style={[...errorLabelStyles, cursor.cursorAuto]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
```

Comment:
```suggestion
                        error={<Text style={[...this.props.errorLabelStyles, cursor.cursorAuto]}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}
```

Reply:
Is in necessary? I think we can just use `this.props.errorLabelStyles` directly
```js
error={<Text style={this.props.errorLabelStyles}>{this.props.translate('attachmentView.failedToLoadPDF')}</Text>}

Reply:
From the `proptype`, `PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object), PropTypes.func])` . I know in our case we are passing an array, but there will be no linter errors if a single object is passed down. I will just keep this as a guard.

Reply:
This will work correctly even if we pass styles as a single object
The previous crash was due to nested arrays, it won't happen if we pass a single object

Reply:
Sure, it should be up-to-date now.